Restore.cmd

```diff
@@ -0,0 +1,8 @@
+@echo off
+set NuGetExe=%~dp0NuGet.exe
+
+echo Restoring packages: Toolsets (this may take some time)
+call %NugetExe% restore -verbosity quiet "%~dp0build\ToolsetPackages\project.json" -configfile "%~dp0nuget.config"
```

Comment:
If there are any spaces in the path which would make quoting the path to `project.json` necessary, then the path to NuGet.exe needs to be quoted as well.